packages/packages/api/src/devices/models/common/source-info.ts

```diff
@@ -1,5 +1,6 @@
 export interface SourceInfo {
+  standardized_type?: string;
```

Comment:
we know all the types here, so this can be improved by making the type an `enum StandardizedType` with values of the strings - then, you can import the type and use it in the API.

see here for an example https://github.com/metriport/metriport/blob/b77b339c6bafc6b8eda57b58bab3edd5fa2fb055/packages/packages/api/src/devices/models/common/metadata.ts#L6